flaskr/logic/adminlgc.py

```diff
@@ -110,10 +111,30 @@ def general_settings(self):
                     else:
                         flash("Interest successfully created")
                         self._interest_handler.insert_interest(current_user.scheme_id, new_interest_form.interest_name.data)
+            
+
+            delete_hobby_form = DeleteHobbyForm(request.form)
+            if delete_hobby_form.hobby.data:
```

Comment:
shouldn't we validate the form first?
`if delete_hobby_form.validate_on_submit():`

Reply:
Probably don't need it as its a selection box